src/mishegos/mutator.c

```diff
@@ -126,6 +129,38 @@ static void rand_opcode(opcode *opc) {
     opc->op[2] = rand_byte();
     break;
   }
+  case 4: { // VEX 2-byte
+    opc->len = 3;
+    opc->op[0] = 0xc5;
+    opc->op[1] = rand_byte();
+    opc->op[2] = rand_byte();
+    break;
+  }
+  case 5: { // VEX 3-byte
+    opc->len = 4;
+    opc->op[0] = 0xc4;
+    opc->op[1] = rand_byte();
+    opc->op[2] = rand_byte();
+    opc->op[3] = rand_byte();
+    break;
+  }
+  case 6: { // XOP
+    opc->len = 4;
+    opc->op[0] = 0x8f;
+    opc->op[1] = rand_byte() | 0x08;
```

Comment:
I'd remove `| 0x08` since it prevents testing some unused maps without bit 3 set, eg. 0x17.

From the AMD vol.3 docs page 71:

> The XOP.map_select value is restricted to
> the range [08h:1Fh]. If the value of the XOP.map_select field is less than 8, the first two bytes of the
> three-byte XOP escape sequence are interpreted as a form of the POP instruction.

Reply:
Agreed!

Reply:
True, changed.